system/gatekeeper/tests/gatekeeper-policies/fixtures/pod-security-v2/host-path-volumes-allow.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: host-path-volumes-allow
+  namespace: test-allow-all
+spec:
+  containers:
+  - name: foo-container
+    image: foo.image-registry.com/test-repo/dummy:20250519150208
+    volumeMounts:
+    - mountPath: /hostfs
+      name: hostfs
+  volumes:
+  - name: hostfs
+    hostPath:
+      path: /
```

Comment:
This should also demonstrate a second container that uses an image not on the allowlist, but is allowed because it does not mount the volume in question.